### PR TITLE
Explicitly designate CommonJS and ESM files with cjs/mjs extensions

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Explicitly use .cjs and .mjs extensions for plain js files (mainly configs) where possible

## Justify why they are needed

Needed for type: module migration in next PR

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/next-i18next.config.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
+const path = require('path')
```
Comment (guilhermespopolin) [inline apps/store/next-i18next.config.js:1]: Should this file be renamed into `next-i18next.config.cjs`?

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/next-i18next.config.js (hunk 1 above)
Comment (alebedev) [reply apps/store/next-i18next.config.js:1]: It broke the build.  I've finally found how to do it in a later PR - was non-trivial

**Comment by alebedev** [comment]
### Merge activity

* **Feb 8, 9:54 AM EST**: @alebedev started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3894).

**Pull request merged**